resource-manager/src/main/java/org/ihtsdo/otf/resourcemanager/ResourceManager.java

```diff
@@ -34,14 +34,14 @@ public class ResourceManager {
 	 *
 	 * @param resourceConfiguration Used to get the resource configuration which details
 	 *                              the local/cloud setting.
-	 * @param resourceLoader   The resource loader associated to the cloud
+	 * @param cloudResourceLoader   The resource loader associated to the cloud
 	 *                              service.
 	 */
-	public ResourceManager(final ResourceConfiguration resourceConfiguration, final ResourceLoader resourceLoader) throws NullPointerException {
+	public ResourceManager(final ResourceConfiguration resourceConfiguration, final ResourceLoader cloudResourceLoader) throws NullPointerException {
```

Comment:
You can use FileSystemResourceLoader as well so it is better to leave the variable name as resourceLoader.